docs/resources/aws_efs_file_system.md

```diff
@@ -0,0 +1,95 @@
+---
+title: About the aws_efs_file_system Resource
+---
+
+# aws\_efs\_file_system
+
+Use the `aws_efs_file_system` InSpec audit resource to test the properties of a single AWS EFS file system.
+
+## Syntax
+
+An `aws_efs_file_system` resource block declares the tests for a single AWS EFS file system by either file system id or creation token.
+
+    describe aws_efs_file_system('fs-12345678') do
+      it                         { should be_encrypted }
+      its('size_in_bytes.value') { should cmp 6144 }
+    end
+
+    describe aws_efs_file_system(creation_token: 'my-token') do
+      its('encrypted')       { should cmp true }
+      its('throughput_mode') { should eq 'bursting' }
+    end
+
+#### Parameters
+
+One of either the EFS file system id or creation token must be provided.
```

Comment:
Perhaps this could be re-worded, there's something about this line I find hard to read. 
Maybe something along these lines
`At least one of the following parameters must be provided:`
OR
`Either the EFS file system id or creation token *MUST* be provided.`

Reply:
Good point.
Do you think **must** should be emphasized in bold or uppercase?

Reply:
Not necessarily. Just an example. happy for it to be lowercase